driverapi/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+set(
+    DRIVERAPI_SOURCES
+    ${CMAKE_SOURCE_DIR}/driverapi/src/librecuda.cpp
```

Comment:
here, again, this should either use CMAKE_CURRENT_LIST_DIR or probably even better just refer to src/librecuda.cpp etc. directly. This is probably more robust than always going from the root.